snuba/clickhouse/optimize/optimize.py

```diff
@@ -394,13 +394,13 @@ def optimize_partitions(
             )
             return
 
+        partition_parts = partition.strip("()").split(",")
+        partition_tuple = (int(partition_parts[0]), partition_parts[1].strip("'"))
+
         args = {
-            "database": database,
-            "table": table,
-            "partition": partition,
+            "partition": partition_tuple,
```

Comment:
actually, i don't quite get how this will be interpolated. it seems that you're stringifying a tuple here?